src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -1472,6 +1476,10 @@ If used, make sure to add the configuration flags just before calling `appToken`
 
 Here's a sample configuration:
 
+```typescript
+
+```
```

Comment:
@ndesai-newrelic Same here, can you provide a Capacitor config example?

Reply:
it is already on the website.